recipe/meta.yaml

```diff
@@ -57,17 +62,33 @@ requirements:
     - scipy >=1.10.1
     - scikit-image >=0.19.0
     - scikit-learn >=1.2
-    - tensorflow >=2.4.0,<2.16
     - tifffile
     - tk >=8,!=8.6.9
     - tqdm
     - zarr
+    # TensorFlow is not available for windows on conda-forge
+    # We have made the dependency optional and point users to an alternative
+    # method. however the conda-forge packaging guidelines disallow depending
+    # on external channels
+    - tensorflow >=2.4.0,<2.16   # [unix]
```

Comment:
hmm, it seems that tensorflow 2.16 was never build for python 3.12 on conda-forge.